.env.example

```
@@ -6,3 +6,5 @@ VITE_DEFAULT_IRC_SERVER_NAME="Local"
 VITE_DEFAULT_IRC_CHANNELS="#lobby,#bots,#test"
 # Optionally hide the server list
 VITE_HIDE_SERVER_LIST=true
+# Optional comma-separated list of trusted media URLs (for chat bridge image proxies like Matterbridge, Matrix bridges)
+VITE_TRUSTED_MEDIA_URLS="https://matterbridge.example.com,https://matrix-media.example.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)